kraken-app/kraken-app-hub/src/test/resources/mock/api-targets/api-target.order.notification.state.change.yaml

```diff
@@ -0,0 +1,33 @@
+---
+kind: kraken.component.api-target
+apiVersion: v1
+metadata:
+  key: mef.sonata.api-target.order.notification.state.change
```

Comment:
Please remove this file if it is useless

Reply:
Thanks. This file has been deleted.